### ISSUE TITLE
[BUG] Coins drop from mobs even when they die of natural causes

### ISSUE THREAD

**Issue by KM-Owner** [PluginBugs/Issues-RPGMoney#46]
**Describe the bug**
Coins drop from mobs even when they die of natural causes.
In earlier versions they had to be killed by the player.

**Expected behavior**
If this is the intended action, please add options to the config

**Info (please complete the following information):**
 - Server version: 1.20.4
 - Client version: 1.20.4
 - Plugin version: RPGMoney 1.2.0

Uploading Minecraft_ 1.20.1 - Multiplayer (3rd-party Server) 2024-01-04 19-34-39.mp4…

**Comment by KM-Owner** [comment]
![image](https://github.com/PluginBugs/Issues-RPGMoney/assets/122387923/4b97793a-03a2-4c10-af13-0fbc56c084ef)

![image](https://github.com/PluginBugs/Issues-RPGMoney/assets/122387923/ad6af132-9e29-4d3f-8557-5f138ac69500)

**Comment by Ivanusshka** [comment]
I agree, the same problem, and another bug with multipliers, a lot of money is falling

**Comment by KM-Owner** [comment]
> I agree, the same problem, and another bug with multipliers, a lot of money is falling

Yes, I have the same problem with multipliers (a lot of money is falling)

**Comment by Ivanusshka** [comment]
It's been 2 weeks, the developer has not fixed the bug

**Comment by Nahalyd** [comment]
Workaround for no killer:
add "only_if_killer_is_a_player: true" to all loots rules

untested workaround for multiplier madness:
did you try not using it or setting them to 1.0 ?

**Comment by TACHY134** [comment]
I only get only_if_killer_is_a_player in the players, but in the mobs I don't eat, sheep, pigs, cows ![image](https://github.com/PluginBugs/Issues-RPGMoney/assets/165131389/ddb4dfa8-49bf-4470-9bcf-47d569a8eba8) 
![Captura de la consola 2](https://github.com/PluginBugs/Issues-RPGMoney/assets/165131389/1cccfa6a-39a0-490e-832a-4f4f77262aa4)

**Comment by Nahalyd** [comment]
It's a kind of issue with this plugin, configuration file rules are not described. So we dont know all possible settings.
We had to test and discover in order to fix an economy issue recently.
(Edits = typo)

**Comment by TACHY134** [comment]
A possible solution?

**Comment by Nahalyd** [comment]
it was our solution, seeing that the property is useable everywhere, and it motivates me to post a review. each loots rule has been patched with this line.

Some players have got 20k during the issue resolution so we have to wipe globally (heavy loss) or regulate (unfair for the honest  ones).

**Comment by kohle** [comment]
The workaround from Nahalyd isn't working for me.

Are there any expected timelines for this big to be fixed?

**Comment by BronzWarq** [comment]
Can we expect this bug to be fixed or should we already be looking for something else?

**Comment by kohle** [comment]
> Can we expect this bug to be fixed or should we already be looking for something else?

@BronzWarq Check out LootConomy, it has the same core functionality and works much better.

**Comment by LoneDev6** [comment]
> Coins drop from mobs even when they die of natural causes.

You have to set `natural_death.enabled: false` on each loot configuration.

```yml
natural_death:
   enabled: false
```

Example:
```yml
iron_golem:
  enabled: true
  entities:
  - IRON_GOLEM
  money: 25-50
  drops_count: '1'
  chance: 100
  natural_death:
    enable: false
  worlds:
  - world
  - world_the_end
  - world_nether
```
